pkg/login/social/social.go

```diff
@@ -174,6 +188,7 @@ func NewOAuthService() {
 				emailAttributeName:   info.EmailAttributeName,
 				emailAttributePath:   info.EmailAttributePath,
 				roleAttributePath:    info.RoleAttributePath,
+				groupMappings:        info.GroupMappings,
```

Comment:
If/when adding this support to all OAuth providers we shouldn't need to do this. Instead groupMappings could possibly be added to `SocialBase` since `newSocialBase` takes `info` as argument. SocialBase could then hold the `extractGroupMappings` method and be used by all OAuth providers.

Reply:
I can add this change when I extend this PR to support the other OAuth providers, if that's something this PR should do.